rfcs/0006-working-in-the-open.md

```diff
@@ -0,0 +1,104 @@
+---
+
+[La version française suit.](#résumé)
+
+# Working in the open
+## Summary
+CDS as an organization has a commitment to working in the open, inspired by our
+peers in the UK, United States, and around the world. As we develop software
+products, coding in the open provides a number of important benefits:
+ * It encourages good practice
+ * It makes collaboration easier
+ * External users can help make it better
+ * Others can learn from our work
+ * It makes it easier to share standards
+ * It improves transparency on government’s work
+ * It clarifies ownership
+ * It helps make government technology seamless
+ * It’s easier to start code in the open than to open a closed repository
+
+This RFC attempts to lay out the _how_ of our open source work.
+
+
+## Motivation
+When we collaborate in the open and publish our data publicly, we can improve
+Government together. Building services more openly and publishing open data,
+simplifies access to government services and information, allows the public to
+contribute, and enables reuse by entrepreneurs, nonprofits, other agencies, and
+the public.
+
+CDS’s commitment to open source helps to normalize its use and creation across
+the Government of Canada. The [GC Digital Standards](https://www.canada.ca/en/government/publicservice/modernizing/government-canada-digital-standards.html)
+endorse the use and development of open source software within the Canadian
+Government.
+
+
+## Proposal
+* For new development choose a permissive license approved by the Open Source
+  Initiative, preferring the MIT License. MIT is the default license for USDS,
+  GDS, 18F, the Ontario Digital Service, and others.
+
+* When adapting existing code from another organization, understand and respect
+  the license terms of the original code. When forking and adapting existing
+  projects, refer to the [set of considerations](https://github.com/canada-ca/open-source-logiciel-libre/blob/master/en/guides/using-open-source-software.md#verify-open-source-software-licence) to
+  understand and apply license terms and conditions from the GC Office of the
+  CIO (OCIO).
+
+* Where possible, avoid using software or packages that are not licensed under
+  an OSI or FSF-approved open source license.
+
+* The [CDS-SNC GitHub organization](https://github.com/cds-snc) is our default
+  code repository. If needed or desired by a partner, a similar organization on
+  an open code hosting site maybe used.
+
+* For each code repository, publish:
+  * A simple and clear code of conduct
+  * Contribution guidelines
+  * License
+  * Security vulnerability disclosure process
+  * Issue and PR templates
+
+  Refer to the [repository configuration](https://github.com/cds-snc/.github)
+  project for examples to copy.
+
+* Adhere to the Official Languages Act by providing bilingual documentation,
+  creating bilingual templates for issues and pull requests, and responding to
+  issues in the language of the issue whenever possible.
```

Comment:
> and responding to issues in the language of the issue whenever possible.

I would break this out into two points: 


* Adhere to the Official Languages Act by providing bilingual documentation,
  creating bilingual templates for issues and pull requests,

* Issues opened in an official language must be responded to in that official language.

I'm not sure we are allowed to put a caveat around language of choice when it comes to interacting with the public, I'm pretty sure OLA mandates that we respond in their language of choice and that their rights to interact in their official language of choice supersedes our rights to respond in our language of choice. 

https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=26160

Reply:
done. Sounds like 6.2.5 is the relevant section, thanks for clarifying.